guides/common/modules/proc_downloading-python-packages-to-a-host-from-a-custom-python-repository.adoc

```diff
@@ -0,0 +1,53 @@
+[id="Downloading_Files_to_a_Host_from_a_{custompythontypeid}_Repository_{context}"]
+= Downloading Files to a Host From a {custompythontypetitle} Repository
+
+You can download files to a client over HTTPS using `curl -O`, and optionally over HTTP if the *Unprotected* option for repositories is selected.
+
+.Prerequisites
+* You have a {custompythontype} repository.
+* You know the name of the python package you want to download to clients from the {custompythontype} repository.
+* To download the python packages over HTTPS to your managed hosts, you require:
++
+. The `katello-server-ca.crt`.
+For more information, see {AdministeringDocURL}Importing_the_Katello_Root_CA_Certificate_admin[Importing the Katello Root CA Certificate] in _{AdministeringDocTitle}_.
+. An Organization Debug Certificate.
+For more information, see {ManagingOrganizationsLocationsDocURL}Creating_an_Organization_Debug_Certificate_managing-organizations-locations[Creating an Organization Debug Certificate] in _{ManagingOrganizationsLocationsDocTitle}_.
+
+.Procedure
+. In the {ProjectWebUI}, navigate to *Content* > *Products*.
+. Select a {customproduct} by name.
+. Select a `python` type repository by name.
+. Optional: Select the *Unprotected* checkbox to access the repository published through HTTP.
+. Copy the *Published At* URL.
```

Comment:
This is missing the final step which the CLI procedure has. IMHO it's fine if you copy and paste but you can also create a snippet. Your call.

Reply:
I've created a snippet for this. Do you think it makes sense to include the snippet twice; for ".Produce" and ".CLI procedure"?

Reply:
Yes, I would include it twice just for completeness.

Reply:
OK, no change required then.